.github/workflows/release-static-production.yml

```diff
@@ -3,15 +3,13 @@ name: 'Release: Static Production'
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - static-production-amp-dev
+  pull_request:
 
 jobs:
   queue:
     runs-on: ubuntu-latest
     outputs:
-      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+      should_skip: ${{ steps.skip_check.outputs.should_skip || steps.check_user.check-result == 'false' }}
```

Comment:
What is this check for?

Reply:
it gets the output of [`this`](https://github.com/ampproject/amp.dev/blob/b42e077300cc035ccfd0c788d750810a44784ad3/.github/workflows/release-static-production.yml#L22-L25). It is checking if the creator has write permission on the repo